### PR TITLE
initial release

### PR DESCRIPTION
**Pull request by aptalca**

### CODE REVIEW

File: root/defaults/authelia-location.conf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Version 2020/05/31 - Changelog: https://github.com/linuxserver/docker-letsencrypt/commits/master/root/defaults/authelia-location.conf
```
Comment (nemchik) [inline root/defaults/authelia-location.conf:1]:
```suggestion
## Version 2020/05/31 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/authelia-location.conf
```

File: root/defaults/authelia-server.conf [hunk 2]
```diff
@@ -0,0 +1,48 @@
+## Version 2020/05/31 - Changelog: https://github.com/linuxserver/docker-letsencrypt/commits/master/root/defaults/authelia-server.conf
```
Comment (nemchik) [inline root/defaults/authelia-server.conf:1]:
```suggestion
## Version 2020/05/31 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/authelia-server.conf
```

File: root/defaults/default [hunk 3]
```diff
@@ -0,0 +1,147 @@
+## Version 2020/05/23 - Changelog: https://github.com/linuxserver/docker-letsencrypt/commits/master/root/defaults/default
```
Comment (nemchik) [inline root/defaults/default:1]:
```suggestion
## Version 2020/05/23 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/default
```

File: root/defaults/jail.local [hunk 4]
```diff
@@ -0,0 +1,57 @@
+## Version 2020/05/10 - Changelog: https://github.com/linuxserver/docker-letsencrypt/commits/master/root/defaults/jail.local
```
Comment (nemchik) [inline root/defaults/jail.local:1]:
```suggestion
## Version 2020/05/10 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/jail.local
```

File: root/defaults/ldap.conf [hunk 5]
```diff
@@ -0,0 +1,92 @@
+## Version 2020/06/02 - Changelog: https://github.com/linuxserver/docker-letsencrypt/commits/master/root/defaults/ldap.conf
```
Comment (nemchik) [inline root/defaults/ldap.conf:1]:
```suggestion
## Version 2020/06/02 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/ldap.conf
```

File: root/defaults/nginx.conf [hunk 6]
```diff
@@ -0,0 +1,105 @@
+## Version 2019/12/19 - Changelog: https://github.com/linuxserver/docker-letsencrypt/commits/master/root/defaults/nginx.conf
```
Comment (nemchik) [inline root/defaults/nginx.conf:1]:
```suggestion
## Version 2019/12/19 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx.conf
```

File: root/defaults/proxy.conf [hunk 7]
```diff
@@ -0,0 +1,32 @@
+## Version 2019/10/23 - Changelog: https://github.com/linuxserver/docker-letsencrypt/commits/master/root/defaults/proxy.conf
```
Comment (nemchik) [inline root/defaults/proxy.conf:1]:
```suggestion
## Version 2019/10/23 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/proxy.conf
```

File: root/defaults/ssl.conf [hunk 8]
```diff
@@ -0,0 +1,48 @@
+## Version 2020/06/17 - Changelog: https://github.com/linuxserver/docker-letsencrypt/commits/master/root/defaults/ssl.conf
```
Comment (nemchik) [inline root/defaults/ssl.conf:1]:
```suggestion
## Version 2020/06/17 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/ssl.conf
```

**Comment by aptalca** [comment]
oh yeah, good eye. Forgot about those

**Pull request merged**